sdk/core/core-http-compat/src/policies/keepAlivePolicy.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import {
+  PipelinePolicy,
+  PipelineRequest,
+  SendRequest,
+  PipelineResponse,
+} from "@azure/core-rest-pipeline";
+
+export const keepAlivePolicyName = "KeepAlivePolicy";
+
+export function createKeepAlivePolicy(): PipelinePolicy {
```

Comment:
nit: can we rename this to `disableKeepAlivePolicy` for clarity?

Reply:
Done